tensornetwork/contractors/custom_path_solvers/nconinterface.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from typing import List, Union, Tuple, Optional
+# pylint: disable=line-too-long
+from tensornetwork.contractors.custom_path_solvers.pathsolvers import full_solve_complete
+
+
+def ncon_solver(tensors: List[np.ndarray],
```

Comment:
hey Glen, what do you think of `ncon_path` instead of `ncon_solver`?

Reply:
I prefer `ncon_solver` as `ncon_path` sounds like it refers to the `con_order` itself. Something like `ncon_path_finder` could be an option.

Reply:
sgtm, let's keep it as it is